### PR TITLE
Added app routing, scss, header and footer

### PR DESCRIPTION
**Pull request by BabkinAV**
### Created basic app structure: 

- `Header` component
- `Footer` component

### Added basic routing between dashboard and home pages

### CODE REVIEW

**Review by TGiulio — APPROVED** [review]

**Review by williamsgqdev — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -3,12 +3,18 @@
...
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
+    <meta name="description" content="The best budget planner" />
```
Comment (patrikglanemann) [inline public/index.html:17]: :D 👍

File: public/index.html [hunk 2]
```diff
@@ -24,7 +30,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Geckspance</title>
```
Comment (patrikglanemann) [inline public/index.html:33]: I think this is a misspelling, it should be Geckspence.

File: src/layout/Header.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { Navbar, Container, Nav } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+
+function Header() {
+  return (
+    <header>
+      <Navbar expand="lg" variant="dark">
+        <Container>
+          <Navbar.Brand as={Link} to="/" className="text-white">
+            Geckspance
```
Comment (patrikglanemann) [inline src/layout/Header.js:11]: Here as well.

**Review by patrikglanemann — CHANGES_REQUESTED** [review]
Good job on creating the basic layout, as well on the favicon, I like it:)
I am used to put components in a component folder rather than in a layout folder, but that is personal taste I guess:D
I left you a comment about the App name, I think it is a misspelling.

**Pull request merged**